test-data/unit/fine-grained.test

```diff
@@ -8901,16 +8901,29 @@ def bar(x):
     # type: () -> None
     pass
 
+def baz(x):
+    # type: () -> None
+    pass
+
+[file c.py]
+def bar(x):
+    # type: () -> None
```

Comment:
What if the error is generated inside a nested function (i.e. not at module top level)?